async_work_thread_safe_function/node-api/package.json

```diff
@@ -7,6 +7,9 @@
   "dependencies": {
     "bindings": "~1.2.1"
   },
+  "engines": {
+    "node": ">= 10.6.0"
+  },
```

Comment:
This should also work with a version of 8.X as well right?  I do understand that might be hard to reflect in "engines" but if possible maybe a comment to explain why it's set to 10.6.0 would be good?

Reply:
or if it is possible to specify a number of ranges, adding one for 8.x would be good.

Reply:
Node 8 not supported, related api is introduced 
 in 10.6.0 https://nodejs.org/dist/latest-v12.x/docs/api/n-api.html#n_api_napi_threadsafe_function

Reply:
It think that is a quirk of our doc's approach. It is part of N-API version 4 and from the N-API Version matrix in https://nodejs.org/dist/latest-v12.x/docs/api/n-api.html#n_api_n_api_version_matrix.  Version 4 is supported by v8.16.0 and above

Reply:
Latest version of the matrix includes v10.16.0 for 4 as well as an FYI, just not backported to 12.x docs.